docs_outdoornav_user_manual/api/api_examples/mission_with_custom_tasks.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: Mission with Custom Tasks
+sidebar_label: Mission with Custom Tasks
+sidebar_position: 3
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+Before being able to run the examples similar to the one explained below, it is reuired that you have written your own custom tasks. See the
+[Custom Tasks](/docs_outdoornav_user_manual/features/custom_tasks) section for information on how to develop tasks for your application.
+
+## The Code
+
+``` python
+#! /usr/bin/env python3
+
+import rospy
+import actionlib
+from cpr_outdoornav_api_examples_lib.coordinate_lat_lon import CoordinateLatLon
+from cpr_outdoornav_api_examples_lib.ros_node import RosNode
+from cpr_outdoornav_api_examples_lib.waypoint import Waypoint
+from cpr_outdoornav_api_examples_lib.mission import Mission

```

Comment:
Similar note as above regarding these examples being private in 0.9.0. Hope to make them public for 0.10.0.